### DISCUSSION TITLE
Feedback for "Class Variance Authority"

### DISCUSSION THREAD

**Discussion by Pmmarteau** [joe-bell/cva#224] [Feedback]
Hi,

I'm trying to build a design system with vite, react, bun, tailwind and I'm using cva. Unfortunately, when using the package in another project, one of my exported component is trying to import ClassProp from class-variance-authority/types, which is leading to an error. If i'm modifying this to class-variance-authority/dist/types, it works, but I can't modify the node modules in production. Do you have any suggestions on how to fix this problem ? 
![Screenshot 2023-09-27 at 18 50 18](https://github.com/joe-bell/cva/assets/45701556/81fedb9d-f810-4abf-9e71-b5d6084ecfe8)

**Comment by joe-bell** [comment]
Please can you try [`cva@beta`](https://cva.style) and let me know if you experience the same issue? Hopefully this should be resolved
